Makefile

```diff
@@ -98,3 +106,28 @@ pipelines: ## Upload pipelines to Concourse
 showenv: ## Display environment information
 	$(eval export TARGET_CONCOURSE=deployer)
 	@concourse/scripts/environment.sh
+
+.PHONY: manually_upload_certs
+CERT_PASSWORD_STORE_DIR?=~/.paas-pass-high
+manually_upload_certs: ## Display environment information
```

Comment:
Inline comment is incorrect here... (unless the upload certs function really does display the environment information)